web/static/js/components/simulator/SmsSimulator.jsx

```diff
@@ -0,0 +1,213 @@
+import React, { Component, PropTypes } from 'react'
+import { withRouter } from 'react-router'
+import * as test from './testMessages'
+
+class SmsSimulator extends Component {
+
+  state = {
+    messages: test.baseMessages()
+  }
+
+  handleUserSentMessage = message => {
+    const msg = { messageBody: message.messageBody, messageType: 'sent' }
+    this.addMessage(msg)
+  }
+
+  handleBotSentMessage = message => {
+    const msg = { messageBody: message.messageBody, messageType: 'received' }
```

Comment:
`sent` & `received` are point-of-view-sensitive.

It's an industry standard to talk about MO/MT (Mobile Originated/Terminated), or AO/AT (Application Originated/Terminated), which better reflects the point-of-view of the exchange.